src/components/Search/Search.tsx

```diff
@@ -32,22 +42,42 @@ const Search = ({ altText = 'Search', useIcon = true }: React.PropsWithChildren<
       </button>
       <form
         data-testid="search-form"
-        className={classnames(
-          `${px}-search__input-wrapper`,
-          { [`${px}-search__input-wrapper--active`]: overlayEnabled },
-          { [`${px}-search__input-wrapper--use-icon`]: useIcon },
-        )}
+        className={classnames(`${px}-search__form`, { [`${px}-search__form--active`]: overlayEnabled })}
       >
-        {useIcon ? (
-          <SearchIcon data-testid={`search-form-icon`} className={`${px}-search__input-wrapper__icon`} />
-        ) : null}
-        <Input
-          className={`${px}-search__input`}
-          alt={altText}
-          hideLabel={true}
-          placeholder={`Type to search`}
-          type="text"
-        />
+        <div
+          className={classnames(`${px}-search__input-wrapper`, { [`${px}-search__input-wrapper--use-icon`]: useIcon })}
+          role="combobox"
+          aria-haspopup="listbox"
+        >
+          {useIcon ? (
+            <SearchIcon data-testid={`search-form-icon`} className={`${px}-search__input-wrapper__icon`} />
+          ) : null}
+          <Input
+            className={`${px}-search__input`}
+            alt={altText}
+            hideLabel={true}
+            placeholder={`Type to search`}
+            type="text"
+            onChange={
+              onSearch
+                ? (e: { target: { value: string } }) => {
+                    console.log('e.target.value', e.target.value);
+                    onSearch(e.target.value);
```

Comment:
did this need to be debounced?

Reply:
I think we decided to debounce on the consumer side which seems smart since only it knows if it's server-side or client-side updates